### ISSUE TITLE
Request: only work for blank tabs

### ISSUE THREAD

**Issue by averissimo** [sblask/webextension-open-tabs-next-to-current#52]
### New Issue Checklist (tick off like this: `[x]`)

- [X] I checked existing issues (open and closed) for possible duplicates
- [X] I can reproduce the problem on the latest stable version (not nightly!)

### What is your browser?

- [X] Firefox
- [ ] Chrome

### What is your operating system?

- [X] Linux
- [ ] Mac
- [ ] Windows

### Description (please include examples/screenshots where applicable)

Add an option to only work when opening blank and unrelated tabs (without *tabOpenerId*)

This way firefox's configurable behavior of maintaining tab opening order can be used in parallel with this.

I've created a fork for this feature to be persistent *(for my needs)*, it still needs the option page if it is to be implemented.

https://github.com/averissimo/webextension-open-tabs-next-to-current

**Comment by JosephCaillet** [comment]
+1

**Comment by sblask** [comment]
I might do it, but don't hold your breath, I don't have much time at the moment.

**Comment by averissimo** [comment]
Totally understand, this is also a problem when opening closed tabs, as the placement goes crazy.

**Comment by sblask** [comment]
Not sure what you mean by opening closed tabs. What's the problem?

**Comment by JosephCaillet** [comment]
I think he mean re opening a previously closed tab (eg using ctrl+shift+t)

**Comment by sblask** [comment]
Their position should not be changed by this add-on though...

**Comment by averissimo** [comment]
I believe it does, I just tested on a clean profile of 59 with only this extension.

1. Open 4 tabs
1. Close #2 
1. Select any tab execpt #1
1. Hit `Ctrl+Shift+t`

**Comment by sblask** [comment]
See #71 for the undo close tab problem. I will close this issue as I don't have time to work on it. Reopen if someone wants to pick it up.
